3_javascript/javascript2/week2/index.html

```diff
@@ -0,0 +1,65 @@
+<head>
+  <title>HyfBay</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="main.css" />
+  <link
+    href="https://fonts.googleapis.com/css?family=Open+Sans:400,800"
+    rel="stylesheet"
+  />
+</head>
+<body>
+  <main>
+    <header>
+      <div class="wrapper">
+        <h1>HyfBay</h1>
+        <h2>- Buy the okay'est products</h2>
+      </div>
+    </header>
+    <section class="table-view-control">
+      <div class="search">
+        <label>What are you looking for?</label>
+        <input type="text" id="searchInput" />
+      </div>
+      <div class="price">
+        <input type="number" id="maxPriceInput" placeholder="Enter max price" />
+      </div>
```

Comment:
where is the javascript file for this?

Reply:
@szoboszlaianna I mistakenly added js script into JS2 week 1 homework and I did bit a mistake in git branch creation.
Now I have corrected homework as more structured wise.
Would it be possible to have PR on this?
Thanks